release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -170,6 +170,11 @@ Previously, you could not specify the subdomain of a route, and the `spec.host`
 
 You can use this enhancement to simplify sharding by enabling a route to have multiple, distinct host names determined by each router deployment that exposes the route.
 
+[id="ocp-4-11-external-dns-operator"]
+==== External DNS Operator is GA for AWS, Azure, and GCP and is TechPreview for AWS GovCloud
```

Comment:
```suggestion
==== External DNS Operator is GA for AWS Route53, Azure DNS, Google Cloud DNS and Infoblox and is TechPreview for BlueCat and AWS Route53 on GovCloud
```

Reply:
to make the heading shorter and crisp, I added this information in the paragraph and changed the heading to  "External DNS Operator is GA in OpenShift Container Platfrom 4.11"